src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import { FaUser } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+
+function Header() {
+  return (
+    <header className="top">
+      <div className="left-header">
+        <h1>Bookstore CMS</h1>
+        <nav className="navigation">
+          <Link to="/">BOOKS</Link>
+          <Link className="cat" to="/categories">CATEGORIES</Link>
+        </nav>
```

Comment:
✅ Have react-router-dom links set up to click on them and visit navigation pages 👍🏻